main/plots.py

```diff
@@ -55,21 +85,70 @@ def create_timeseries_plot(start_date: datetime, end_date: datetime) -> figure:
     )
     plot.yaxis.axis_label = "Value"
     plot.xaxis.axis_label = "Date"
-    plot.line(
-        "index",
-        "Effort",
-        source=source,
-        line_width=2,
-        color="firebrick",
-        legend_label="Project effort",
+    for trace in traces:
+        plot.line(
+            "index",
+            trace["col_name"],
+            source=source,
+            line_width=2,
+            color=trace["colour"],
+            legend_label=trace["legend_label"],
+        )
+    plot.legend.click_policy = "hide"  # hides traces when clicked in legend
+    return plot
+
+
+def create_capacity_planning_plot(start_date: datetime, end_date: datetime) -> figure:
+    """Generates all the time series data for the capacity planning plot.
+
+    Includes all business days between the selected start and end date, inclusive of
+    the start date. Time for the effort (aggregated over all projects) and
+    capacity (aggregated over all users) are calculated.
+
+    Args:
+        start_date: datetime object representing the start of the plotting period
+        end_date: datetime object representing the end of the plotting period
+
+    Returns:
+        Bokeh figure containing timeseries data.
+    """
+    source = calculate_capacity_planning_traces(start_date, end_date)
+
+    # provide info needed to plot as dictionary for each trace
+    traces = [
+        {"col_name": "Effort", "colour": "firebrick", "legend_label": "Project effort"},
+        {"col_name": "Capacity", "colour": "navy", "legend_label": "Capacity"},
+    ]
+    plot = create_timeseries_plot(
+        source=source, title="Project effort and team capacity over time", traces=traces
     )
-    plot.line(
-        "index",
-        "Capacity",
+
+    return plot
+
+
+def create_cost_recovery_plot() -> figure:
+    """Creates the cost recovery plot for the last year.
+
+    Provides an overview of team capacity over the past year and the project charging.
+
+    Returns:
+        Bokeh figure containing cost recovery data.
+    """
+    source = calculate_cost_recovery_traces()
+
+    # provide info needed to plot as dictionary for each trace
+    traces = [
+        {
+            "col_name": "Cost recovery",
+            "colour": "gold",
+            "legend_label": "Charge per day",
```

Comment:
Given that we are not showing `money` but `capacity` all the time, I think this should be more like `Capacity recovered` or `Capacity used`.

```suggestion
            "legend_label": "Capacity used",
```